### PR TITLE
:seedling: introduce release phases

### PR DESCRIPTION
**Pull request by djzager**
Also provides a tentative plan for the Konveyor v0.4 release.

### CODE REVIEW

File: releases/release-0.4/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Konveyor v0.4
+
+## Summary (Tentative)
+
+The 0.4 release cycle is as follows:
+
+- **Tuesday 2 January 2024**: Release begins
```
Comment (savitharaghunathan) [inline releases/release-0.4/README.md:7]: should we move this by a week and start on jan 9th? Allow time for folks coming back from holidays and catch up?

**Comment by savitharaghunathan** [comment]
issue: https://github.com/konveyor/community/issues/71

**Review by savitharaghunathan — APPROVED** [review]
/lgtm

This is a great list of important dates to remeber/adhere to with a lot of flexibility in terms of cutting alpha/beta/rc. Thanks @djzager :)

**Comment by savitharaghunathan** [comment]
+1 on the phases description. Its clear and makes sense.

**Review by aufi — APPROVED** [review]

**Review by dymurray — APPROVED** [review]

**Pull request merged**